build/Makefile

```diff
@@ -40,8 +40,8 @@ SEPOLICY_DIR_EL6=$(INTERMEDIATE_DIR)/selinux.el6
 
 ifeq ($(ULINUX),1)
 # Doesn't matter what version of SSL/Ruby we use to compile or link the OMI plugin
```

Comment:
What are we building here that doesn't require a specific SSL or Ruby?

Reply:
Seems like the OMI input plugin, please confirm.

Reply:
Yes, this is for OMI input plugin.